deploy/gcloud/Dockerfile.consumer

```diff
@@ -0,0 +1,36 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Use an official Go runtime as a parent image
+FROM golang:1.13
```

Comment:
Might be interesting to look into using one container to build `amppkg`, and then running it from a separate more minimal container. This reduces the container size and perhaps has some security benefits as well.

An example is at https://firebase.google.com/docs/hosting/cloud-run#containerize

Reply:
Thanks @ithinkihaveacat.  If you look at ./gcloud_up.sh.  I actually build 2 versions of the docker image (one for consumer and another for renewer) and upload both to the gcloud container registry and tag it with a version tag.  Not sure if this is what you're trying to point out.  I've discussed with @jeffjose the possibility of running things in cloud-run, but the samples I've seen doesn't allow multiple builds and deployments using cloud-run at this time. Thanks for the feedback!

Reply:
I don't think that's quite what I was suggesting. Since `amppkg` is a self-contained binary, it can *run* on a very minimal (~5MB) image. Of course it can't be built on such an image, but Docker provides a way to build on one image, and then copy the binary to another, something like:

```docker
FROM golang as builder
RUN go get -d github.com/ampproject/amppackager/cmd/amppkg
WORKDIR /go/src/github.com/ampproject/amppackager/cmd/amppkg
RUN CGO_ENABLED=0 GOOS=linux go build -v -o amppkg
FROM alpine
COPY --from=builder /go/src/github.com/ampproject/amppackager/cmd/amppkg/amppkg /amppkg
CMD ["/amppkg"]
```

There's probably no pressing need to use this process to create images like this right now, but it might be good to do so in the future, especially if the same amppkg container will be downloaded and shared by multiple users. Looks like the difference is around 2GB versus 50MB. (The alpine OS image itself is 5MB but `amppkg` is >40MB!)

Reply:
Ah got it, makes sense. Will consider it for this PR or perhaps a follow-on PR.

Reply:
Multi-staged builds, that's what they are called - https://docs.docker.com/develop/develop-images/multistage-build/